pkgs/top-level/unixtools.nix

```diff
@@ -224,9 +224,9 @@ let
       darwin = pkgs.darwin.basic_cmds;
     };
     xxd = {
-      linux = pkgs.vim.xxd;
-      darwin = pkgs.vim.xxd;
-      freebsd = pkgs.vim.xxd;
+      linux = pkgs.tinyxxd.xxd;
+      darwin = pkgs.tinyxxd.xxd;
+      freebsd = pkgs.tinyxxd.xxd;
```

Comment:
@rhelmot Could you check that this builds and runs on FreeBSD? (Thanks for all your work on NixBSD; I really want to try it out sometime.)

Reply:
It works!